shiny/ui/_navs.py

```diff
@@ -1089,13 +1091,20 @@ def layout(self, nav: Tag, content: Tag) -> TagList:
             if self.fillable:
                 content_div = as_fillable_container(as_fill_item(content_div))
         else:
+            tab_content = contents
+            if self._is_page_level:
+                # TODO: This could also be applied to the non-sidebar page layout above
```

Comment:
`TODO-future`?

```suggestion
                # TODO-future: This could also be applied to the non-sidebar page layout above
```

Reply:
Maybe? I didn't because I didn't want to imply we'll do it, I just wanted to leave a note that someone would see if they came around here to bring `card_image()` out of experimental.

Reply:
It's not going to be fixed now, in this PR. So I don't mind the `TODO-future`.